Code/training_common_utils.py

```diff
@@ -163,6 +168,7 @@ def training_loop(
         with torch.no_grad():
             loss_validation = []
             for (images, labels) in validation_loader:
+                images = images.squeeze(0)
```

Comment:
strange.. never had to do this previously. though i may not gotten the additional dimension somehow...